packages/orbit-components/src/primitives/IllustrationPrimitive/types.d.ts

```diff
@@ -7,9 +7,11 @@ import type * as Common from "../../common/types";
 
 export type Size = "extraSmall" | "small" | "medium" | "large" | "display";
 
+// Interface content is fetched from /src/Illustration/TYPESCRIPT_TEMPLATE.template
```

Comment:
You can't write comments in here… Anything written in these files will be overwritten on build.

Reply:
lol